src/django/api/serializers/v1/production_locations_serializer.py

```diff
@@ -1,36 +1,54 @@
 from rest_framework import serializers
 from api.serializers.v1.opensearch_error_list_builder  \
     import OpenSearchErrorListBuilder
+from api.serializers.v1.opensearch_common_validators. \
+    countries_validator import CountryValidator
 from api.serializers.v1.opensearch_common_validators. \
     number_of_workers_validator import NumberOfWorkersValidator
 from api.serializers.v1.opensearch_common_validators. \
     percent_of_female_workers import PercentOfFemaleWorkersValidator
 from api.serializers.v1.opensearch_common_validators. \
     coordinates_validator import CoordinatesValidator
+from api.views.v1.utils import COMMON_ERROR_MESSAGE
 
 
 class ProductionLocationsSerializer(serializers.Serializer):
-
+    # These params are checking considering serialize_params output
+    size = serializers.IntegerField(required=False)
     number_of_workers_min = serializers.IntegerField(required=False)
     number_of_workers_max = serializers.IntegerField(required=False)
     percent_female_workers_min = serializers.FloatField(required=False)
     percent_female_workers_max = serializers.FloatField(required=False)
     coordinates_lat = serializers.FloatField(required=False)
     coordinates_lon = serializers.FloatField(required=False)
+    country = serializers.ListField(
+        child=serializers.CharField(required=False),
+        required=False
+    )
+    sort_by = serializers.ChoiceField(
+        choices=['name', 'address'],
+        required=False
+    )
+    order_by = serializers.ChoiceField(
+        choices=['asc', 'desc'],
+        required=False
+    )
 
     def validate(self, data):
         validators = [
             NumberOfWorkersValidator(),
             PercentOfFemaleWorkersValidator(),
-            CoordinatesValidator()
+            CoordinatesValidator(),
+            CountryValidator(),
         ]
 
         error_list_builder = OpenSearchErrorListBuilder(validators)
         errors = error_list_builder.build_error_list(data)
 
         if errors:
+            # TODO: Pass error msg to the Rollbar here
```

Comment:
Can you pass an error message to the Rollbar in the scope of this task?

Reply:
This is a placeholder for future improvements. There are multiple kind of error messages that might appear here. We need to disscuss which of them stakeholders want to see in the Rollbar dashboard and depict this in a separate ticket.

Reply:
Let's discuss whether it should be passed to Rollbar at all. The error is clearly described and returned to the user with a 400 status code. It is not something that works in a weird way on our side.